### PR TITLE
Сreate test for the service step execution sleep

### PR DESCRIPTION
**Pull request by tatarynovdima**

### CODE REVIEW

File: src/test/java/executor/service/service/StepExecutionSleepTest.java [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+    @Mock
+    private WebDriver webDriver;
+
+    @InjectMocks
+    private StepExecutionSleep stepExecution;
+
+    private StepDTO stepDTO;
+
+    @BeforeEach
+    void setUp() {
```
[the remark this answers is not in the document]
Comment (tatarynovdima) [reply src/test/java/executor/service/service/StepExecutionSleepTest.java]: Thanks i use public for all test methods

[the remark this answers is not in the document]
Comment (artemkedyk) [reply src/test/java/executor/service/service/StepExecutionSleepTest.java]: and the same for class:)

[the remark this answers is not in the document]
Comment (tatarynovdima) [reply src/test/java/executor/service/service/StepExecutionSleepTest.java]: Thank you, @Serhx4 and @artemkedyk i added changes

**Review by AlexandrTokarenko — APPROVED** [review]

**Review by Serhx4 — APPROVED** [review]
Looks good!
Just a reminder: please, don't forget to use Squash and Merge :)

**Review by artemkedyk — APPROVED** [review]

**Pull request merged**